### PR TITLE
doc: a series of changes for mosly minor typos, language tweaks

### PR DESCRIPTION
**Pull request by salewski**
This PR contains changes that fix minor spelling or word-choice typos, adjusts the string quote chars in the example config file to allow them to parse as valid TOML, and adds a few links to mentioned projects.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,56 +4,57 @@
...
-configured in the registries.conf file as follows:
+configured in the 'registries.conf' file as follows:
 
 ```
-unqualified-search-registries=[“registry.fedoraproject.org”, “docker.io”]
+unqualified-search-registries=['registry.fedoraproject.org', 'docker.io']
 
 [aliases]
-“fedora”=”registry.fedoraproject.org/fedora”
+'fedora'='registry.fedoraproject.org/fedora'
```
Comment (saschagrunert) [inline README.md:30]: I prefer double quotes `"` in toml, but that's a non blocking nit.

File: README.md [hunk 2]
```diff
@@ -65,3 +66,8 @@ your case on why your shortname should replace the existing short names.
 ## Contact
 
 - IRC: #[containers](irc://irc.freenode.net:6667/#containers) on freenode.net
+
+
```
Comment (saschagrunert) [inline README.md:70]:
```suggestion
```

**Review by saschagrunert — APPROVED** [review]
LGTM, just two nitty nits.

File: README.md [hunk 3]
```diff
@@ -65,3 +66,8 @@ your case on why your shortname should replace the existing short names.
...
+[podman-gh]:      https://github.com/containers/podman  "GitHub: containers/podman"
+[docker-cli-gh]:  https://github.com/docker/cli         "GitHub: docker/cli"
+[gnu-bash]:       https://www.gnu.org/software/bash/    "GNU Bash (www.gnu.org)"
```
Comment (vrothberg) [inline README.md:73]: I don't think bash is a good contact.

**Review by vrothberg — COMMENTED** [review]
Thanks for contributing!

File: README.md (hunk 1 above)
Comment (salewski) [reply README.md]: I updated it to use double quotes in my latest rebase.

File: README.md (hunk 2 above)
Comment (salewski) [reply README.md]: Fixed in my latest rebase.

File: README.md (hunk 3 above)
Comment (salewski) [reply README.md]: In my latest rebase I removed the link to the gnu.org bash site.

**Review by saschagrunert — APPROVED** [review]
Thanks, LGTM!

**Comment by salewski** [comment]
@saschagrunert @vrothberg Thanks both for your feedback. I've incorporated all of your suggestions into my latest rebase. Please take a look when you can.

**Review by vrothberg — COMMENTED** [review]
LGTM, @rhatdan PTAL

**Pull request merged**